libs/util-prohibited-data/src/index.ts

```diff
@@ -2,8 +2,11 @@ import { toLower } from 'lodash';
 import { Address } from 'viem';
 
 import prohibitedAddresses from './lib/prohibited-addresses.json';
+import prohibitedCountries from './lib/prohibited-countries.json';
 
 export const isAddressProhibited = (address: Address | undefined) => {
```

Comment:
just realised we copy-pasted this function in almost all the apps, since you've touched this file already do you mind removing all that and update imports to this lib? 

<img width="505" alt="image" src="https://github.com/user-attachments/assets/36e20ff1-e3b0-4827-b299-95c5af693ec5">